server/sonar-server/src/main/java/org/sonar/server/measure/ws/ComponentTreeAction.java

```diff
@@ -275,8 +281,8 @@ private static ComponentTreeWsRequest toComponentTreeWsRequest(Request request)
     List<String> metricKeys = request.mandatoryParamAsStrings(PARAM_METRIC_KEYS);
     checkArgument(metricKeys.size() <= MAX_METRIC_KEYS, "Number of metrics keys is limited to %s, got %s", MAX_METRIC_KEYS, metricKeys.size());
     ComponentTreeWsRequest componentTreeWsRequest = new ComponentTreeWsRequest()
-      .setBaseComponentId(request.param(PARAM_BASE_COMPONENT_ID))
-      .setBaseComponentKey(request.param(PARAM_BASE_COMPONENT_KEY))
+      .setBaseComponentId(request.param(DEPRECATED_PARAM_BASE_COMPONENT_ID))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "setBaseComponentId"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)